autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -68,6 +69,15 @@ public void validateStopStartEntitlementAndDisableIfNotEntitled(Cluster cluster)
         }
     }
 
+    public void validateImpalaScheduleBasedScalingEntitlement(String clusterCrn, CloudPlatform cloudPlatform, String stackName) {
```

Comment:
Shouldn't this invocation happen from one of the Controllers ? -i.e. the API call which attempts to enable / disable impala autoscaling on the Periscope side should be making this check.

See additional notes in CronTimeEvaluator.